s4/str_printing_tools.rb

```diff
@@ -0,0 +1,31 @@
+# MLS
+text = <<MLS
+  Hola,
+  este es un texto de prueba
+el cual respeta la disposicióna
+      que tengan todas las líneas.
+Gracias.
+MLS
+puts text; puts
+
+# \"" ''
```

Comment:
When should you use double quotes?

Reply:
When you want to interpolate the String or plan to use escape sequences like `\n`.